src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7352,7 +7352,7 @@ components:
       description: >
         ⚠️ An object describing high level information about a domain for use in the snapshot builder
 
-    SnapshotBuilderGetConceptsResponse:
+    SnapshotBuilderConceptsResponse:
```

Comment:
Should a ticket be opened up to change `DataRepo` types to follow the new type names?

Reply:
Good point! I will change it in my terra ui PR!